content/index.ko.md

```diff
@@ -58,29 +59,31 @@ footer: >
 {% include_cached toc.html type="end" %}
 {:/}
 
-## 상황속 접근성 {#context}
+## 상황에 따른 접근성 {#context}
 
 <blockquote class="pull">
   <p>웹의 힘은 보편성에 있습니다.<br />
     장애에 상관없이 모두가 접근할 수 있다는 것이 가장 중요한 부분입니다.</p>
   <footer><cite>팀 버너스리, W3C 디렉터 및 Wrold Wide Web의 창시자</cite></footer>
 </blockquote>
 
-웹은 기본적으로 하드웨어, 소프트웨어, 언어, 장소, 능력에 제약없이 모든 사람들이 사용할 수 있도록 설계되었습니다. 웹이 이 목표를 달성할 때 다양한 청력, 움직임, 시력, 인지 능력을 가진 사람들이 접근할 수 있습니다.
+웹은 기본적으로 하드웨어, 소프트웨어, 언어, 장소, 능력에 제약없이 모든 사람들이 사용할 수 있도록 설계되었습니다. 웹이 이 목표를 달성할 때 비로소 다양한 청력, 움직임, 시력, 인지 능력을 가진 사람들이 웹에 접근할 수 있습니다.
 
-많은 사람들이 물리적 세상에서 커뮤니케이션과 상호작용하는 데에 마주하는 어려움을 웹이 없애주기 때문에 웹에서의 장애의 영향이 급격하게 변합니다. 그러나 웹사이트, 어플리케이션, 기술이나 도구가 안 좋게 설계되면 사람들이 웹을 사용하는 데에 배제되는 장벽을 만들게 될 수도 있습니다. 
+이와 같이 웹은 많은 사람들이 물리적 세상에서 의사소통과 상호작용하는 데에 마주하는 어려움을 웹이 없애주기 때문에 웹에서의 장애의 영향이 급격하게 변합니다. 그러나 웹사이트, 어플리케이션, 기술이나 도구가 불충분하게 설계되면, 그것들은 사람들이 웹을 사용하는 것을 배제하는 장벽을 만들 수도 있습니다. 
+
+
+**고품질의 웹사이트와 웹 도구를 만들고자 하는 개발자와 조직에게 접근성은 필수적입니다. 접근성은 제품과 서비스를 이용하는 데 있어 배제되는 사람들이 생기지 않도록 하기 때문입니다.**
```

Comment:
@remibetin 
Thanks for the comment, but there are no repeated phrases in my edits. If it helps, here's the Google Translate version of my translation:

>In this way, the impact of disability on the Web is dramatically changing because the Web eliminates the difficulties many people face in communicating and interacting in the physical world. But when websites, applications, technologies or tools are poorly designed, they can create barriers that exclude people from using the Web.
**Accessibility is essential for developers and organizations who want to create high-quality websites and web tools. Accessibility ensures that no one is excluded from using products and services.**

Based on your clarification, I've revised the translation as follows:

고품질의 웹사이트와 웹 도구를 만들고자 하며, 동시에 그들의 제품 및 서비스 사용에서 사람들을 배제하지 않으려는 개발자와 조직에게 웹 접근성은 필수적입니다.